Cargo.toml

```diff
@@ -13,7 +13,7 @@ log = "0.4"
 parking_lot = { version = "0.9", optional = true }
 parity-scale-codec = { version = "1.0.3", optional = true, default-features = false, features = ["derive"] }
 num = { package = "num-traits", version = "0.2", default-features = false }
-hashmap_core = { version = "0.1.10", default-features = false }
+hashbrown = { version = "0.6" }
```

Comment:
We need to compile with `default-features = false`. `finality-grandpa` is `no-std` compatible.

Reply:
`hashbrown` supports no_std by default.

Reply:
Oh, I didn't know that. Thanks :)